setup-jest.js

```diff
@@ -1,3 +0,0 @@
-jest.setTimeout(20000);
```

Comment:
Why was this removed?

Reply:
The value is now set in the main Jest config file as `testTimeout`.

Reply:
Ok 👍